src/include/cell_inline.h

```diff
@@ -845,6 +848,28 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
         break;
     }
 
+    /*
+     * Unpack any fast-truncate information. Note that there is no way to write fast-truncate
```

Comment:
Should this just skip the fast truncate information, rather than unpacking it?

I guess it's a question of whether information that was generated by a newer release should be parsed out and maybe, therefore, alter decisions made at runtime.

It feels as though it is going to be hard to test that combination (i.e: run with a newer version, generate fast truncate information, downgrade, read fast truncate information, ensure correct decisions are being made).

So, I'd be inclined towards ignoring the information as early as possible, rather than using it to populate data structures.

Reply:
I agree with Alex. Instead of saving the unpacked data into the unpack_addr, just save them into a local variable and ignore them.

Reply:
Yes, I don't believe it's possible to simply skip the information - I think it could disrupt other information that is being unpacked as it would affect the position of the pointer. I can definitely unpack it into a local WT_PAGE_DELETED structure rather than into a structure inside the unpack_addr.

Reply:
Should we move this logic into a dedicated function?

Reply:
@quchenhao Would there be any specific advantages of having a separate function for this? We're only unpacking the information once.

Reply:
Just to make the code cleaner.